Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectBuilder/MSBuildBuildConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using UnityEngine;
+
+namespace Microsoft.Build.Unity
+{
+    [Serializable]
+    public sealed class MSBuildBuildConfiguration
```

Comment:
>MSBuildBuildConfiguration [](start = 24, length = 25)

Minor nit. Naming this Configuration is a little bit confusing because "Configuration" is a well defined property in MSBuild. Could alternatively be MSBuildBuildSettings.

Reply:
I renamed it to MSBuildBuild**Profile**.